config/base/software.nix

```
@@ -21,43 +21,44 @@ in
 
   environment.systemPackages = with pkgs; [
     # external package formats
-    flatpak
+    flatpak                      # Good software utility to use if you can't find what you want at nixpkgs
     # snapd N/A yet
 
     # sys apps
-    conf-tool
-    nixNodePackage
+    conf-tool                    # Config Tool
+    nixNodePackage               # NodeJS pre-installed
```

not really. actually:

```suggestion
    nixNodePackage               # hack to get nix-node-package to be available without downloading it at every eval
```